### PR TITLE
Get artifact by final name when user run azure-functions:package directly

### PR DESCRIPTION
**Pull request by Flanker32**
Get artifact by final name when user run `azure-functions:package` directly, fixes https://github.com/microsoft/azure-maven-plugins/issues/1212

### CODE REVIEW

File: azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java [hunk 1]
```diff
@@ -377,7 +379,23 @@ protected JsonObject readHostJson() {
 
     protected void promptCompileInfo() throws AzureExecutionException {
         Log.info(String.format("Java home : %s", System.getenv("JAVA_HOME")));
-        Log.info(String.format("Artifact compile version : %s", Utils.getArtifactCompileVersion(project.getArtifact().getFile())));
+        Log.info(String.format("Artifact compile version : %s", Utils.getArtifactCompileVersion(getArtifact())));
+    }
+
+    private File getArtifact() throws AzureExecutionException {
```
Comment (shenqianjin) [inline azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java:385]: how about getArtifactFile ?

**Review by shenqianjin — APPROVED** [review]
LGTM~

File: azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java [hunk 2]
```diff
@@ -90,8 +91,9 @@
 
     private static final BindingEnum[] FUNCTION_WITHOUT_FUNCTION_EXTENSION =
         {BindingEnum.HttpOutput, BindingEnum.HttpTrigger};
-    public static final String EXTENSION_BUNDLE_ID = "Microsoft.Azure.Functions.ExtensionBundle";
-    public static final String SKIP_INSTALL_EXTENSIONS_BUNDLE = "Extension bundle specified, skip install extension";
+    private static final String EXTENSION_BUNDLE_ID = "Microsoft.Azure.Functions.ExtensionBundle";
+    private static final String SKIP_INSTALL_EXTENSIONS_BUNDLE = "Extension bundle specified, skip install extension";
+    private static final String CAN_NOT_FIND_ARTIFACT = "Can not find the maven artifact, please run mvn package first.";
```
Comment (andxu) [inline azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java:96]: Cannot

Comment (andxu) [reply azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/PackageMojo.java:96]: run 'mvn package'

**Pull request merged**